pkg/controller/controller_ref_manager.go

```diff
@@ -18,13 +18,13 @@ package controller
 
 import (
 	"fmt"
-	"strings"
```

Comment:
go fmt shouldn't mess with the imports once they are split into groups. You need to add back the line below that you are removing. stdlib imports should be separated from k8s imports.